packages/compass-import-export/src/modules/export.js

```diff
@@ -523,6 +522,7 @@ export const startExport = () => {
       return dispatch(onError(err));
     } finally {
       track('Export Completed', {
+        type: exportData.aggregation ? 'aggregation' : 'query',
```

Comment:
That's the only modification we agreed to do for the existing event to not overcomplicate it

Reply:
For my understanding, is the complication on the analysis side or our telemetry event maintenance side in Compass? Cool

Reply:
I don't really have a direct answer. From my perspective if we wanted to extend this event with aggregation specific metadata, it would also make sense to split it into separate events and this would probably require us to separate the other events too for consistency.